lib/matplotlib/figure.py

```diff
@@ -3061,10 +3061,18 @@ def savefig(self, fname, *, transparent=None, **kwargs):
             if transparent:
                 kwargs.setdefault('facecolor', 'none')
                 kwargs.setdefault('edgecolor', 'none')
+                for subfig in self.subfigs:
+                    subfig.set_facecolor('none')
```

Comment:
I don't think this gets reversed when the savefig is completed?

Reply:
Oopsie, good catch.